dm/docs/RFCS/20220308_support_dump_files_store_in_s3.md

```diff
@@ -0,0 +1,26 @@
+# Support dump files store in S3
+
+## Background
+
+When DM synchronizes data at the full stage, it requires a dynamic space which is related to upstream database instance to dump data files from upstream, and the size of this space will be a factor that has to be considered when customer use resources on the cloud. In order to simplify the effect of this factor, we need support dump files store in S3 which is a dynamic store on the cloud.
+
+### User config change
+
+We still use loader's `dir` to config S3 url, but it should be noted that loader's `import-mode` should not be `loader` and it can be `sql` now, because we use Lightning to load files in S3. 
+```
+loaders:
+  global:
+    pool-size: 16
+    dir: s3://dmbucket/dump?region=us-west-2&endpoint=http://127.0.0.1:8688&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
```

Comment:
do we have doc for these query paramaters such as `access_key`?

Reply:
we can get some message from here.
https://docs.aws.amazon.com/AmazonS3/latest/userguide/MakingRequests.html